### DISCUSSION TITLE
Trojan:Win32/Wacatac.B!ml in win11

### DISCUSSION THREAD

**Discussion by rforcen** [ELENA-LANG/elena-lang#671] [General]
i get a Trojan:Win32/Wacatac.B!ml virus warning in w11 when executing a simple app:

```
import system'routines;
import extensions;

public program()
{
   int s:=0;
   console.writeLine("init...");
   for (int i := 0; i < 100000000; i++) { s+=i; };
   console.printLine("s:", s);   
}
```

**Comment by arakov** [comment]
Hi,
it is false positive. Unfortunately happens time to time for different examples. It is heuristic detection and often baseless. 
Just restore it from quarantine list. You can run the virus scan to be safe.

**Reply by rforcen** [reply]
👍
